messages/src/msg_fields/protocols/coordinate_mediation/keylist_query.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+use typed_builder::TypedBuilder;
+
+use crate::{decorators::thread::Thread, msg_parts::MsgParts};
+
+pub type KeylistQuery = MsgParts<KeylistQueryContent, KeylistQueryDecorators>;
```

Comment:
We don't have this across the board, but I just though it would be nice to leave a comment to the message spec (with fragment in the url), for example here

https://github.com/hyperledger/aries-rfcs/blob/main/features/0211-route-coordination/README.md#key-list-query

Reply:
Done